LogMonitor/docs/README.md

```diff
@@ -251,3 +252,101 @@ CMD "c:\\windows\\system32\\ping.exe -n 20 localhost"
 ```
 
 The Process Monitor will stream the output for `c:\windows\system32\ping.exe -n 20 localhost`
+
+## Log Format Customization
+
+### Description
+By default, logs will be displayed in JSON format. However, users can change the log format to either `XML` or their own `custom` defined format.
+
+To specify the log format, a user needs to configure the `logFormat` field in `LogMonitorConfig.json` to either `XML`, `JSON` or `Custom` <em>(the field value is not case-insensitive)</em>
+<br>For `JSON` and `XML` log formats, no additional configurations are required. However, the `Custom` log format, needs further configuration. For custom log formats, a user needs to specify the `customLogFormat` at the source level.
+
+### Custom Log Format Pattern Layout
+To ensure the different field values are correctly displayed in the customized log outputs, ensure to wrap the field names within modulo operators (%) and the field names specified matches the correct log sources' field names.
+
+For example: `%Message%, [%TimeStamp%]`<br>
```

Comment:
The opening and closing square brackets might be confusing? No?  Maybe change this part [%TimeStamp%] to %TimeStamp% ?

Reply:
@bobsira thank you for pointing this out. The square brackets have been removed to avoid confusion. A user can add them if need be.